valgrind_support/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
+include(EkatCreateUnitTest)
 
 add_executable(serial_hw serial_hw.cxx)
 add_executable(mpi_hw mpi_hw.cxx)
 
+EkatCreateUnitTest(mpi_valg_catch_test mpi_catch_test.cpp)
```

Comment:
I think you might have forgotten to include this file in the commit maybe?

Reply:
Good catch! Fixed.